kvariants/src/lib.rs

```diff
@@ -34,15 +33,15 @@ pub static KVARIANTS: Lazy<HashMap<char, KVariant>> = Lazy::new(|| {
     //   㓻 (U+34FB)	sem	    剛 (U+525B)
     //   ...
     //
-    let tsv = include_str!("../../../../dictionaries/txt/chinese/kVariants.tsv");
+    let dictionary = include_str!("../dictionaries/compressed/kVariants.csv");
     let mut reader =
-        csv::ReaderBuilder::new().delimiter(b'\t').has_headers(false).from_reader(tsv.as_bytes());
+        csv::ReaderBuilder::new().has_headers(false).from_reader(dictionary.as_bytes());
```

Comment:
using `include_bytes!` instead of `include_str!` upper should allow us to remove this:
```suggestion
        csv::ReaderBuilder::new().has_headers(false).from_reader(dictionary);
```

Reply:
Fixed in b696feb. The `let dictionary: &[u8]` annotation was added otherwise the `Read` trait bound wouldn't work for `&[u8; 137909]` from `include_bytes!`.